mne/decoding/receptive_field.py

```diff
@@ -203,13 +218,37 @@ def fit(self, X, y):
 
         # Create input features
         X, y = self._delay_and_reshape(X, y)
+
         self.estimator_.fit(X, y)
-        del X, y
         coef = get_coef(self.estimator_, 'coef_')  # (n_targets, n_features)
-        shape = [n_feats, len(self.delays_)]
+        shape = [n_feats, n_delays]
         if self._y_dim > 1:
             shape.insert(0, -1)
         self.coef_ = coef.reshape(shape)
+
+        # Inverse-transform model weights
+        if self.patterns:
+            if isinstance(self.estimator_, TimeDelayingRidge):
+                cov_ = self.estimator_.cov_ / float(n_times * n_epochs - 1)
+                y = y.reshape(-1, y.shape[-1], order='F')
+            else:
+                X = X - X.mean(0, keepdims=True)
+                cov_ = np.cov(X.T)
+            del X
+
+            # Inverse output covariance
+            inv_Y = 1. / float(n_times * n_epochs - 1)
```

Comment:
this should go in an `else` clause because it's not used in the `y.ndim == 2 and y.shape[1] != 1` case